src/utils/bot.rs

```diff
@@ -13,3 +13,22 @@ pub fn load_configuration() -> Configuration {
 pub async fn get_data_lock(ctx: &serenity::Context) -> Arc<RwLock<Data>> {
     ctx.data.read().await.get::<Data>().unwrap().clone()
 }
+
+fn is_404(err: &serenity::http::HttpError) -> bool {
+    if let Some(serenity::http::StatusCode::NOT_FOUND) = err.status_code() {
+        return true;
+    }
+    false
```

Comment:
```suggestion
    matches!(err.status_code(), Some(serenity::http::StatusCode::NOT_FOUND))
```

Reply:
Thank you. I didn't know that was a thing.